src/classes/RD2_ValidationService_TEST.cls

```diff
@@ -774,7 +774,164 @@ private with sharing class RD2_ValidationService_TEST {
         );
     }
 
+    /***
+    * @description Verifies that RD is inserted successfully when Status is mapped
+    */
+    @isTest
+    private static void shouldSuccessfullyInsertRDWhenStatusIsMapped() {
+        RD2_EnablementService_TEST.setRecurringDonations2Enabled();
+
+        RD2_StatusMapper_TEST.Stub mapperStub = new RD2_StatusMapper_TEST.Stub()
+            .overrideDefaultMapping(
```

Comment:
The override should be replaced with `withStatusMapping(RD2_Constants.STATUS_ACTIVE, RD2_Constants.STATUS_ACTIVE)`.